src/main/java/no/nav/tag/finnkandidatapi/tilgangskontroll/TokenUtils.java

```diff
@@ -23,14 +23,20 @@ public TokenUtils(OIDCRequestContextHolder contextHolder) {
     }
 
     public String hentOidcToken() {
-        return contextHolder.getOIDCValidationContext().getToken(ISSUER_ISSO).getIdToken();
+        if (erInnloggetNavAnsattMedAzureADToken()) {
```

Comment:
kan oneline metoden for å slippe duplisering(eller putte issuer i en variabel) :
return contextHolder.getOIDCValidationContext().getToken(erInnloggetNavAnsattMedAzureADToken() ? ISSUER_ISSO :ISSUER_OPENAM).getIdToken();

Reply:
Ønsker vi en oneliner her? Det er litt vanskelig å skjønne hva koden gjør om alt er på samme linje...
Er det noe annet vi kunne gjort for å gjøre det mer leselig?